nautobot/ipam/models.py

```diff
@@ -1092,9 +1096,9 @@ def clean(self):
         if self._namespace is None:
             raise ValidationError({"parent": "Either a parent or a namespace must be provided."})
 
+        closest_parent = self._get_closest_parent()
         # Validate `parent` can be used as the parent for this ipaddress
         if self.parent:
-            closest_parent = self._get_closest_parent()
```

Comment:
What did this change do?

Reply:
`IPAddressForm` do not pass in `parent`, making `if self.parent` check ignored in the `clean` method, but in the `save` method an exception might occur when `self._get_closest_parent()` is called leading to an unhandled exception.

Hence the change, now any exception that occurs would be gracefully handled

Reply:
@jathanism was there a reason that `IPAddress.parent` is nullable and we weren't validating it in clean unless a parent was provided?

Reply:
That's because `parent` is automatically generated, 
also to avoid silently discarding the `parent` if is still provided we validate, normally parent should not be provided